pvlib/bifacial/loss_models.py

```diff
@@ -0,0 +1,140 @@
+import numpy as np
+import pandas as pd
+
+
+def power_mismatch_deline(
+    rmad,
+    model=(0, 0.142, 0.032),
+    fill_factor: float = None,
+    fill_factor_reference: float = 0.79,
+):
+    r"""
+    Estimate DC power loss due to irradiance non-uniformity.
+
+    This model is described for bifacial modules in [1]_, where the backside
+    irradiance is less uniform due to mounting and site conditions.
+
+    The power loss is estimated by a polynomial model of the Relative Mean
+    Absolute Difference (RMAD) of the cell-by-cell total irradiance.
+
+    Use ``fill_factor`` to account for different fill factors between the
+    trained model and the module of interest. Specify model's fill factor with
+    ``fill_factor_reference``.
+
+    .. versionadded:: 0.11.1
+
+    Parameters
+    ----------
+    rmad : numeric
+        The Relative Mean Absolute Difference of the cell-by-cell total
+        irradiance. [Unitless]
+        Check out the *Notes* section for the equation to calculate it from the
+        bifaciality and the front and back irradiances.
+
+    model : float collection or numpy.polynomial.polynomial.Polynomial, default ``(0, 0.142, 0.032)``
```

Comment:
```suggestion
    coefficients: float collection or numpy.polynomial.polynomial.Polynomial, default ``(0, 0.142, 0.032)``
```

Would this be better?  pvlib usually uses `model` for string names, like `model='perez'`.